company/team/index.md

```diff
@@ -2,7 +2,12 @@
 
 This page contains brief bios of our team. Teammates may also have a personal documentation page in this directory that is named according to their Sourcegraph email address (e.g. you@sourcegraph.com -> you.md).
 
-To add yourself to this page, copy the following template, paste it at the end of this file, and make it about yourself!
+To add yourself to this page:
+1. Accept GitHub's email invite to the Sourcegraph org (and then make sure that you are added to the appropriate teams).
+1. Go to the bottom right of this page and click "Edit this page." 
```

Comment:
I might add a bit here, something like this to line 7-->

"this will direct you to GitHub's file editor user interface. If you'd like to learn more about how to edit our handbook on GitHub check out [this page](https://about.sourcegraph.com/handbook/people-ops/onboarding/git_intro)."

Linking to this page, right in this part of the handbook helps new users not to make the mistake of pushing directly,  rather than doing a pull request which is sourcegraph best practice because of how continuous integration is configured (i totally made this mistake 👍)